packages/logger/src/transports/console.ts

```diff
@@ -14,16 +14,34 @@ export class ConsoleTransport implements ITransport {
 
   protected _consoleWrite: LogWriter;
 
+  protected _ignoreRules: RegExp[] = [];
+
   constructor(
     protected _options?: {
+      /**
+       * By default the console transport is disabled.
+       */
       enabled?: boolean;
+      /**
+       * By default all logs are displayed.
+       */
       filter?: string;
+      /**
+       * The storage to save the log.
+       */
       storage?: Storage;
+      /**
+       * Ignore logs that match the rules.
+       */
+      ignoreRule?: string[];
     }
   ) {
     this._consoleWrite = createLogWriter({
       storage: this._options?.storage,
     });
+    this._ignoreRules = (this._options?.ignoreRule ?? []).map(
+      (rule) => new RegExp(rule)
```

Comment:
does we need use  new RegExp here, is that will meet the issue of escape text

maybe just keep original nad use includes? or support pass regex in the `ignoreRule: string|Regex[]`

Reply:
sure.